crypto/curve25519/asm/x25519-asm-x86_64.S

```diff
@@ -27,6 +27,10 @@
 /* OS X's C ABI prefixes functions with underscore. */
 #define C_ABI(x) _ ## x
 #define HIDDEN .private_extern
+#elif defined(__MINGW32__) || defined(__MINGW64__)
+/* MinGW build support */
+#define C_ABI(x) x
+#define HIDDEN .globl
```

Comment:
@briansmith Hi, no. I tried it and if we just #define the HIDDEN macro into nothing then  it doesn't compile.

Looking in another win32 port project i found that they are defining HIDDEN into .globl too. So i wrote it this way.
https://github.com/git-for-windows/MINGW-packages/blob/2a4f9271d7b17498b46b73c11f249ca56b93dfc3/mingw-w64-clang/0061-libunwind-add-support-for-mingw-w64.patch#L92

I don't have any better idea yet.